src/libraries/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -369,7 +369,7 @@ internal static bool GenerateSerializerToStream(XmlMapping[] xmlMappings, Type?[
 
                     writer.Write("[assembly:");
                     writer.Write(typeof(XmlSerializerVersionAttribute).FullName);
-                    writer.Write("(");
+                    writer.Write('(');
                     writer.Write("ParentAssemblyId=");
```

Comment:
can join them:
```suggestion
                    writer.Write("(ParentAssemblyId=");
```

Reply:
@kasperk81 done

Reply:
I disagree with this change, it's sacrificing clarity for a minor and theoretical performance improvement. Could you please revert?

Reply:
few lines below there is `WriteLine(")]")`, how is this any different?

Reply:
@kasperk81 WriteLine  no ovveride char

Reply:
@eiriktsarpalis I wonder what the performance improvement is. I do not understand?

Reply:
@eiriktsarpalis Write(string constant) vs Write(string constant)  + Write(char constant)